File: Javaaa/program/Kendaraan.java
class Kendaraan {
    private String merk;
    private String nama;
    private String warna;
    private int tahun;

    public Kendaraan() {
    }

    public Kendaraan(String merk, String nama, String warna, int tahun) {
        this.merk = merk;
        this.nama = nama;
        this.warna = warna;
        this.tahun = tahun;
    }

    public void setMerk(String merk) {
        this.merk = merk;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public void setWarna(String warna) {
        this.warna = warna;
    }

    public void setTahun(int tahun) {
        this.tahun = tahun;
    }

    public String getMerk() {
        return this.merk;
    }

    public String getNama() {
        return this.nama;
    }

    public String getWarna() {
        return this.warna;
    }

    public int getTahun() {
        return this.tahun;
    }
}

File: python/program/Kendaraan.py
class Kendaraan:
    def __init__(self, merk, nama, warna, tahun):
        self.__merk = merk
        self.__nama = nama
        self.__warna = warna
        self.__tahun = tahun
    
    def set_merk(self, merk):
        self.__merk = merk
    
    def set_nama(self, nama):
        self.__nama = nama
    
    def set_warna(self, warna):
        self.__warna = warna
    
    def set_tahun(self, tahun):
        self.__tahun = tahun
    
    def get_merk(self):
        return self.__merk
    
    def get_nama(self):
        return self.__nama
    
    def get_warna(self):
        return self.__warna
    
    def get_tahun(self):
        return self.__tahun